UPGRADE.md

```diff
@@ -51,17 +51,18 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta1 to Unreleased]
 ### [shopsys/project-base]
-- [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
+- *(optional)* [#497 adding php.ini to image is now done only in dockerfiles](https://github.com/shopsys/shopsys/pull/497)
     - you should make the same changes in your repository for the php.ini configuration files to be added to your Docker images
+        - remove all volumes from `docker-compose.yml` templates that include `php-ini-overrides.ini`
+            - remove them also from your local `docker-compose.yml`
+        - add `COPY project-base/docker/php-fpm/php-ini-overrides.ini /usr/local/etc/php/php.ini` into `docker/php-fpm/ci/Dockerfile`
     - from now on, you will have to rebuild your Docker images (`docker-compose up -d --build`) for the changes in the php.ini file to apply
 - [#494 Microservices webserver using nginx + php-fpm](https://github.com/shopsys/shopsys/pull/494)
     - execute `docker-compose pull` to pull new microservice images and `docker-compose up -d` to start newly pulled microservices
-    - url addresses to microservices have changed, you need to upgrade url address provided in `app/config/parameters.yml`  
+    - url addresses to microservices have changed, you need to upgrade url address provided in `app/config/parameters.yml`
         - update parameter `microservice_product_search_url` from `microservice-product-search:8000` to `microservice-product-search`
         - update parameter `microservice_product_search_export_url`, from `microservice-product-search-export:8000` to `microservice-product-search-export`
-- [#502 - fixed acceptance tests (loading DB dump)](https://github.com/shopsys/shopsys/pull/502)
-    - when you upgrade `codeception/codeception` to version `2.5.0`, you have to change parameter `populate` to `true`
-      in `tests/ShopBundle/Acceptance/acceptance.suite.yml`
+- *(optional)* when you upgrade `codeception/codeception` to version `2.5.0`, you have to change parameter `populate` to `true` in `tests/ShopBundle/Acceptance/acceptance.suite.yml`
```

Comment:
is this OK?
in the change above is *(optional)* [#497 adding php.ini to image is now done only in dockerfiles
but here - [#502 - fixed acceptance tests (loading DB dump)](https://github.com/shopsys/shopsys/pull/502) disappeared.

Reply:
yes, I felt no need for the link